openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -467,13 +467,8 @@ def post(self, request, *args, **kwargs):
         if response:
             return response
 
-        if 'is_account_recovery' in request.GET:
-            self.post_reset_login = True
```

Comment:
should we also remove `post_reset_login` from `__init__` of `PasswordResetConfirmWrapper`? I think it is not being used anymore and its default value is also `False`

Reply:
Done